build_all_ui_files.py

```diff
@@ -0,0 +1,46 @@
+BUILDER = r'C:\Users\OMi32458\Workspace\mantid\external\src\ThirdParty\lib\python2.7\pyuic4.bat'
```

Comment:
@EltayebAhmed We are going to have to rework this at some point so it both works on other machines and on other platforms. We probably want to look at using the `PyQt4.uic` http://pyqt.sourceforge.net/Docs/PyQt4/designer.html#the-uic-module. This PR is getting quite large though so let's leave it as a future issue.